invesalius/gui/task_navigator.py

```diff
@@ -1527,6 +1527,18 @@ def OnMenuSetTarget(self, evt):
         else:
             wx.MessageBox(_("No data selected."), _("InVesalius 3"))
 
+    def OnMenuRemoveTarget(self, evt):
```

Comment:
When the guiding interface is active and you remove a target, the guiding interface button is disabled but remains selected (blue square) on Windows 11. In this event should this should be fixed (or where the guiding interface button functions are).

Reply:
Thanks! This is an old issue. I solved using an internal wx function. take a look at 686135f